tests/devices/test_default_qubit_torch.py

```diff
@@ -20,9 +20,15 @@
 import pytest
 import cmath
 import math
+import functools
 
 torch = pytest.importorskip("torch", minversion="1.8.1")
 
+#TODO: make this togglable via the command line
+use_cuda = torch.cuda.is_available()
+if use_cuda:
+    torch.tensor = functools.partial(torch.tensor, device='cuda')
```

Comment:
@antalszava will this force `default.qubit.torch` to _always_ be in GPU mode if a GPU is available? What happens if the user wants to use a CPU? Wouldn't the issue still occur, just in the opposite direction 🤔 (e.g., user wants CPU, output of the QNode is GPU)

Reply:
Yep, hence the todo on line 27 :slightly_smiling_face: The idea would be to make this optionally be specified when running the test suite.